NitroxClient/MonoBehaviours/Gui/MainMenu/MainMenuMultiplayerPanel.cs

```diff
@@ -218,8 +225,9 @@ private void OnAddServerButtonClicked()
             serverNameInput = serverNameInput.Trim();
             serverHostInput = serverHostInput.Trim();
             serverPortInput = serverPortInput.Trim();
-            AddServer(serverNameInput, serverHostInput, serverPortInput);
-            CreateServerButton($"{Language.main.Get("Nitrox_ConnectTo")} <b>{serverNameInput}</b>\n{serverHostInput}:{serverPortInput}", serverHostInput, serverPortInput);
+            Guid token = Guid.NewGuid();
```

Comment:
Let's generate server side and send back.

Reply:
If the attacker wishes to send corrupted data to the server as their tracking protocol, it would only be useful to tracking hackers, and while we have done that (thank you idiot bad guys) by tracing their bad implementation of RSA keys, I do not see the reason to care, since we not trying to solve the problem but move it, security through obscurity I like to call it.